### PR TITLE
[testnet] Fix a woopsie in loadtest.yaml

### PR DESCRIPTION
**Pull request by perryjrandall**
There was a hanging end, a recent change removed a conditional without testing

Test Plan: this was blocking devnet release, so i unblock devnet release

We really need better automated testing around this

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4035)

### CODE REVIEW

File: terraform/helm/testnet-addons/templates/loadtest.yaml [hunk 1]
```diff
@@ -60,7 +60,6 @@ spec:
             - --target-tps={{ .config.target_tps }}
             - --duration={{ .config.duration }}
             - --txn-expiration-time-secs={{ .config.txn_expiration_time_secs }}
-            {{- end }}
             env:
```
Comment (perryjrandall) [inline terraform/helm/testnet-addons/templates/loadtest.yaml:63]: unexpected

File: .github/workflows/python-unit-test.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        uses: actions/setup-python@v4
+
+      - name: Install Python Dependencies
+        run: pip install -r testsuite/requirements.txt
+
+      - name: Should run tests
+        run: >
+          python testsuite/determinator.py
+          changed-files
+          --github-key SHOULD_RUN
```
Comment (rustielin) [inline .github/workflows/python-unit-test.yaml:34]: nit: github-output-key

**Review by rustielin — APPROVED** [review]
stamp to unblock, probably want one other reviewer though

**Review by sherry-x — APPROVED** [review]
Amazing, thank you!!

**Pull request merged**